leshan-core/src/main/java/org/eclipse/leshan/core/request/exception/RequestRejectedException.java

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v1.0 which accompany this distribution.
+ *
+ * The Eclipse Public License is available at
+ *    http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ *    http://www.eclipse.org/org/documents/edl-v10.html.
+ *
+ * Contributors:
+ *     Bosch Software Innovations GmbH
+ *               - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.leshan.core.request.exception;
+
+/**
+ * Exception indicating that the message was rejected by the LWM2M client.
+ */
+public class RequestRejectedException extends RuntimeException {
+
+    private static final long serialVersionUID = 1L;
+
+    public RequestRejectedException() {
+        super();
+    }
+
+    /**
+     * @param message the message returned by the server or <code>null</code> if the server did not return a message
```

Comment:
I think there is not message payload for RST message in CoAP spec. Even if it exists the Californium API don't expose it. So we can remove this parameter.